src/html5_parser/parser/adoption_agency.rs

```diff
@@ -340,43 +341,43 @@ mod test {
 
         parser
             .active_formatting_elements
-            .push(ActiveElement::Node(NodeId(3)));
+            .push(ActiveElement::Node(3.into()));
         parser
             .active_formatting_elements
-            .push(ActiveElement::Node(NodeId(7)));
+            .push(ActiveElement::Node(7.into()));
 
         assert_eq!(
             parser
-                .find_furthest_block_idx(NodeId(1))
+                .find_furthest_block_idx(1)
                 .expect("furthest element not found"),
             1
         );
         assert_eq!(
             parser
-                .find_furthest_block_idx(NodeId(2))
+                .find_furthest_block_idx(2)
```

Comment:
We're still working with `NodeId`, but our interface is now a little more ergonomic.